community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
I don't see an issue with putting miscellaneous items in a common bucket, these are noteworthy points and I don't see a better placeholder for them as yet. As we build this document out we'd be able to segment these better.


But, if you feel strongly about it how do you propose I segment the points? What could be an appropriate heading?